src/View/Helper/AbstractHelper.php

```diff
@@ -565,7 +565,7 @@ protected function hasAllowedPrefix(string $attribute): bool
      *
      * @internal
      */
-    protected function translateLabel(string|int $label): string|int
+    protected function translateLabel(string $label): string
```

Comment:
This will fix psalm issues in #260

Reply:
Is there a reason int was allowed in the first place?

Reply:
see https://github.com/laminas/laminas-form/pull/256#discussion_r1442113315

Reply:
I think removing 'int' causes a BC Break. For example, if we want to generate pagination and we use only numbers:

    $value = 10;

    /** @var \Laminas\Form\Element\Select $field */
    $field = $this->get('items_per_page');

    $values = [];
    for ($i = 1; $i < 4; $i++) {
        $val = $i * $value;

        $values[$val] = $val;
    }

    $field->setValueOptions($values);

Error message:
> Laminas\Form\View\Helper\AbstractHelper::translateLabel(): Argument # 1 ($label) must be of type string, int given, called in .../laminas/laminas-form/src/View/Helper/FormSelect.php on line 201

https://github.com/laminas/laminas-form/blob/ffdf00a4315b950e47983253a259ad1490ccf6bd/src/View/Helper/FormSelect.php#L201 

The solution is the need to always remember to cast all values to string:

    $values[$val] = (string) $val;

Shouldn't this change be accompanied by a change: https://github.com/laminas/laminas-form/pull/256?

Reply:
I'm with you here. A `(string)` cast should be used for all calls, or `translateLabel` should be altered to accept `scalar` and perform the cast there.

Conversely, all `labels` should be string values regardless, so I do think it's up to the user to convert labels to strings first, but that doesn't stop this from being a BC break based on existing behaviour.

Reply:
See #261